src/Inventory.java

```diff
@@ -0,0 +1,275 @@
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+/**
+ * Created by melmo on 12/1/16.
+ */
+public class Inventory {
+    private int CURRENT_ITEM_ID = 1000;
+    private String CURRENT_USER = null;
+    private HashMap<Integer, Item> inventory = new HashMap<>();
+    private HashMap<String, String> users = new HashMap<>();
+    private Scanner input = new Scanner(System.in);
+    private SimpleDateFormat DF = new SimpleDateFormat("dd/M/yyyy");
+
+    public void login(){
+        System.out.println("What is your username?");
+        String name = getValidStr(this.input.nextLine());
+        String pass = users.get(name);
+        while (pass == null){
+            System.out.println("\nThat username does not exist. Please enter a valid username.");
+            name = getValidStr(this.input.nextLine());
+            pass = users.get(name);
+        }
+        System.out.println("\nWhat is your password?");
+        String password = getValidStr(this.input.nextLine());
+        while (!pass.equals(password)){
+            System.out.println("\nPlease enter the correct password.");
+            password = getValidStr(this.input.nextLine());
+        }
+        CURRENT_USER = name;
+        System.out.format("\n<%s> is logged in.", CURRENT_USER);
+        whatYouWant();
```

Comment:
This method should be refactored so that `login` is not acting as a gate keeper for the program to run.  It should just check the login and the main method should be in charge of the flow of the program which checks the choices.